src/main/java/mate/jdbc/util/ConnectionUtil.java

```
@@ -0,0 +1,32 @@
+package mate.jdbc.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String JDBC_DRIVER = "com.mysql.cj.jdbc.Driver";
+    private static final String USER_NAME = "root";
+    private static final String PASSWORD = "120696";
+    private static final String URL = "jdbc:mysql://localhost:3306/manufacturer_db";
+
+    static {
+        try {
+            Class.forName(JDBC_DRIVER);
+        } catch (ClassNotFoundException e) {
+            e.printStackTrace();
```

let's throw an exception instead

---

done